msprime/stats.py

```diff
@@ -389,20 +391,6 @@ def Y2(self, sample_sets, windows):
         return self.Y2_vector(sample_sets, windows, indices=[(0, 1)])
 
     def Y3(self, sample_sets, windows):
-        """
```

Comment:
This was redundant, but shouldn't it have some documentation? Or is there some trick to documenting both in the same place?

Reply:
Uh, yeah meant to comment. I will look into trick. I was deleting for now as the content should duplicate `Y3_vector`